components/src/molecules/rating/Rating.spec.js

```diff
@@ -0,0 +1,22 @@
+import { mount } from '@vue/test-utils'
+import ARating from './Rating.vue'
+
+describe('Rating', () => {
+  it('has default structure', () => {
+    const wrapper = mount(ARating)
+
+    expect(wrapper.is('div')).toBe(true)
+    expect(wrapper.classes()).toContain('a-rating')
+    expect(wrapper.classes().length).toBe(1)
+  })
+
+  it('renders slot text when passed', () => {
+    const wrapper = mount(ARating, {
+      slots: {
+        item: `<span data-test="item">x</span>`
+      }
+    })
+
+    expect(wrapper.find('.a-rating > button > span').exists()).toBe(true)
+  })
```

Comment:
It'll be nice to have at least one test for checking the logic of rating, like clicking on 3rd star will return selected rate 3 or something like that - just to know if selecting on rating options works properly